src/iconsManifest/supportedExtensions.ts

```diff
@@ -5386,6 +5386,12 @@ export const extensions: IFileCollection = {
       filename: true,
       format: FileFormat.svg,
     },
+    {
+      icon: 'vyper',
+      extensions: [],
```

Comment:
```suggestion
      extensions: ['vyi'],
```

Reply:
I don't believe this is a valid extension for vyper?  Is there an example of this usage?

Reply:
Ah, I found it, looks like it is for interfaces: https://docs.vyperlang.org/en/latest/interfaces.html#standalone-interfaces